test/es6/expo.js

```diff
@@ -34,18 +34,52 @@ var tests = [
             var a = 2;
             a**= 4;
             assert.areEqual(Math.pow(2,4), a, "Exponentiation assignment works as intended");
+        
             var b = -2;
             b **= -4;
             assert.areEqual(Math.pow(-2,-4), b,"Exponentiation assignment works as intended");
+
+            var c = -3;
+            var d = 0;
+            d = ++c**3;
+            assert.areEqual(Math.pow(-2,3), d,"Exponentiation assignment works as intended with preincrement operator");

```

Comment:
These descriptions are vague and require the reader to comprehend the test code to understand what is being tested and what is expected.  Would be better to say something clear about what is being tested, like `preincrement operator has higher precedence than exponentiation operator`.